### ISSUE TITLE
Tests failing after official Docker Distribution registry (`registry:2`) removed `htpasswd`

### ISSUE THREAD

**Issue by chanseokoh** [GoogleContainerTools/jib#2537]
Our integration tests run
```
docker run --rm --entrypoint htpasswd registry:2 -Bbn username password
```

On my desktop where it cached an old version of `registry:2`, the command works.
```
$ docker run --rm --entrypoint htpasswd registry:2 -Bbn username password
username:$2y$05$RO8TSDjphyaccxuztK6L.uQrJ0csk.YpAhfvS7cm2d5Z/cI3PAZm6

$ docker images
REPOSITORY                               TAG                 IMAGE ID            CREATED             SIZE
registry                                 2                   708bc6af7e5e        4 months ago        25.8MB
```

However, the new `registry:2` image they created 27 hours ago fails, as it doesn't have `htpasswd`.
```
$ docker run --rm --entrypoint htpasswd registry:2 -Bbn username password
docker: Error response from daemon: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"htpasswd\": executable file not found in $PATH": unknown.
$ docker images
REPOSITORY               TAG                  IMAGE ID            CREATED             SIZE
registry                 2                    2d4f4b5309b1        27 hours ago        26.2MB
```

Here's a sample [Kokoro build failure log](https://source.cloud.google.com/results/invocations/0921a6bc-ce79-4cbe-8fa3-04846176ea21/log).

**Comment by chanseokoh** [comment]
If we use it only to run `htpasswd`, then maybe we can just switch to any image providing `htpasswd`.

Alternatively, we could pin to an older `registry:2` image (only if the image is for having `htpasswd`).

**Comment by chanseokoh** [comment]
Fixed by #2538.
